src/platform/nrfconnect/wifi/WiFiManager.h

```diff
@@ -88,16 +88,9 @@ class Map
 class WiFiManager
 {
 public:
-    enum WiFiRequestStatus : int
-    {
-        SUCCESS    = 0,
-        FAILURE    = 1,
-        TERMINATED = 2
-    };
-
     using ScanResultCallback = void (*)(const NetworkCommissioning::WiFiScanResponse &);
-    using ScanDoneCallback   = void (*)(WiFiRequestStatus);
-    using ConnectionCallback = void (*)();
+    using ScanDoneCallback   = void (*)(const wifi_status &);
+    using ConnectionCallback = void (*)(int);
```

Comment:
Why not using `wifi_conn_status` instead of `int`?

Reply:
Agree.. not sure, why I used `int` :)